src/app/services/getdata.service.ts

```diff
@@ -0,0 +1,15 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { environment } from 'src/environments/environment';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class GetdataService {
+
+  constructor(private http: HttpClient) { }
+
+  getData(){
```

Comment:
- Create an interface to define a model `ng g interface models/model-name`(In this case model-name should be the kind of object the api is going to give us (eg: books)
- Inside the interface define the scheme of the object you're going to get from the api
- After defining the interface do this: https://medium.com/better-programming/modeling-typescript-interfaces-to-support-angular-http-requests-add452e15b7f